CONTRIBUTING.md

```diff
@@ -651,3 +651,79 @@ Names of files and directories should be in lowercase, with dashes between words
 
   As an exception, an explicit conditional expression with null can be used when fixing a important bug without triggering a mass rebuild.
   If this is done a follow up pull request _should_ be created to change the code to `lib.optional(s)`.
+
+## I opened a PR, how do I get it merged?
+[i-opened-a-pr-how-do-i-get-it-merged]:#i-opened-a-pr-how-do-i-get-it-merged
+
+In order for your PR to be merged, someone with merge permissions on the repository needs to review and merge it.
+Because the group of people with merge permissions is mostly a collection of independent unpaid volunteers who do this in their own free time, this can take some time to happen. It is entirely normal for your PR to sit around without any feedback for days, weeks or sometimes even months. We strive to avoid the latter cases of course but the reality of it is that this does happen quite frequently. Even when you get feedback, further feedback may take similarly long. Don't be intimidated by this and kindly ask for feedback again every so often. If your change is good it will eventually be merged at some point.
+
+There are some things you can do to help speed up the process of your PR being merged though. In order to speed the process up, you need to know what needs to happen before a committer will actually hit the merge button. This section intends to give a little overview and insight of what happens after you create your PR.
+
+### The committer's perspective
+
+PRs have varying quality and even the best people make mistakes. It is the role of the committer team to assess whether any PR's changes are good changes or not. In order for any PR to be merged, at least one committer needs to be convinced of its quality enough to merge it.
+
+Committers typically assess three aspects of your PR:
+
+1. Whether the change's intention is necessary and desirable
+2. Whether the code quality of your changes is good
+3. Whether the artefacts produced by the code are good
+
+If you want your PR to get merged quickly and smoothly, it is in your best interest to help convince committers in these three aspects.
+
+### How to help committers assess your PR
+
+For the committer to judge your intention, it's best to explain why you've made your change. For trivial changes like version updates, you can omit this part of course because the intention is obvious but for anything more nuanced or even major version upgrades, it helps if you explain the background behind your change a bit. E.g. if you're adding a package, explain what it is and why it should be in Nixpkgs.
```

Comment:
This is somewhat contrary to other parts of the documentation where we suggest to include a changelog url for version upgrades. I quite like this suggestion.

Reply:
Hm, changelogs are a good point though. They're useful to have but I never bother to include them myself... ^^'